db/schema.rb

```diff
@@ -14,7 +14,6 @@
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "citext"
-  enable_extension "pg_stat_statements"
```

Comment:
Este cambio no es necesario.

Chistosamente lei de esta extension hoy 😂 

https://www.citusdata.com/blog/2019/02/08/the-most-useful-postgres-extension-pg-stat-statements/

por si gustas ver para que funciona, sin embargo, no es necesario que la habilites en tu maquina, solo evitemos commitear cambios al schema cuando no son necesarios 😄